ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -349,6 +349,14 @@ objects:
                         --key /tmp/artifacts/installer/tls/journal-gatewayd.key \
                         --url "https://${bootstrap_ip}:19531/entries?_SYSTEMD_UNIT=${service}.service"
                 done
+                if ! whoami &> /dev/null; then
+                  if [ -w /etc/passwd ]; then
```

Comment:
this is read-only file

Reply:
made writable in https://github.com/openshift/origin/pull/22592